### ISSUE TITLE
Pattern for representing lesions and their shape required

### ISSUE THREAD

**Issue by matentzn** [obophenotype/upheno#302]
Some POs such as PLANP and MP represent lesions. PLANP in particular needs to represent lesions and their very particular shape. In MP, some lesions are represented as process phenotypes (example:[ skin lesions](https://www.ebi.ac.uk/ols/ontologies/mp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMP_0001212)). 

How do we represent, for example, a **_star shaped lesion of the tail_**? 

@srobb1, @sbello, @dosumis, @pnrobinson

**Comment by srobb1** [comment]
I was thinking about this:

 1. The shape inheres in the lesion  
 2.  and the shaped lesion inheres in the tail

```
'has part' some 
    (   
        ('star shaped' and ('inheres in' some 'lesion')   )
	and ('inheres in' some 'tail' )
         and ('has modifier' some abnormal)
   )
```

**Comment by sbello** [comment]
Double 'inheres in' is not allowed, maybe substitute 'occurs in' for the second one? Based on the BFO definition I think this is correct if 'lesion' is a process term as in the MP example. Where is 'lesion' coming from?

**Comment by sbello** [comment]
Nope, reading the NCIT definition the lesion sounds like a material entity not a process. So, we're back to entity located on/in another entity. Closest pattern I can find is molecular entity in a location. This has the pattern '('inheres_in' some (%s and ('part_of' some %s)))' I think that should work here. So we would end up with

'has part' some 
    (   
        ('star shaped' and ('inheres in' some (lesion and ('part of' some tail)) )
	
         and ('has modifier' some abnormal)
   )

**Comment by pnrobinson** [comment]
Is there anything wrong with modelling lesions as material entities (I think this is much more accurate than process)?

**Comment by sbello** [comment]
No problem with modeling lesions as material entities. I also thinks this makes more sense than the current MP pattern which strikes me as too broad. The issue was occurs in BFO species process to entity so we can't use that relationship. The last pattern above is appropriate for entity to entity.

**Comment by dosumis** [comment]
Probably better interpreted as a material entity rather than a process.  This fits with the quality - a process can’t be ‘star shaped’.  

You should therefore use: 

star-shaped and inheres_in some (leision that part_of some tail)

Or perhaps 

star-shaped and inheres_in some (leision that located_in some tail)

as we typically use part_of to record canonical anatomy.  

I think OK to use NCIT term for now.

**Comment by srobb1** [comment]
@sbello Thanks! This looks like it would work great! I am going to try it out.

**Comment by matentzn** [comment]
If the lesions are lesions and do not need to be further classified (i.e. different subtypes of lesions), I suggest to specify the pattern a bit (fix the lesion): 
```
'has_part' some (%s and ('inheres_in' some ('lesion' and ('part_of' some %s))) and ('qualifier' some 'abnormal'))
```
There is always a tension between the flexibility of a pattern and possible issues of interoperability.  3 variables should IMHO be used only in very specific circumstances (relational qualities etc).

**Comment by dosumis** [comment]
I think the generalized pattern we need to agree on here is

Q inheres_in some ( {pathological entity} & R some {anatomical entity})

Should R be part_of or located_in.  I favour the latter but this should be decided in co-ordination with Mondo.

If we don't want to subtype lesions (apart from by where they are - which we are specifying here using a compositional entity), then I agree with Nico that we should make a lesion pattern.

**Comment by srobb1** [comment]
I created a pull request #303 with a suggested pattern to move forward. Any comments and/or corrections on the pattern are welcome and appreciated. I made a note in the comments of the pull request that there are questions about the best relation to use, part_of or located_in.  

Sofia
